packages/plugin-flex/src/commands/flex/plugins/upgrade-plugin.ts

```diff
@@ -115,6 +140,13 @@ export default class FlexPluginsUpgradePlugin extends FlexPlugin {
    * @override
    */
   async doRun(): Promise<void> {
+    const uiVersion = this.flexUIVersion;
+    if (uiVersion >= 2 && !this._flags['flex-ui-2.0']) {
```

Comment:
`if (this.flexUIVersion >= 2 ...)` and remove `const uiVersion = this.flexUIVersion;`